src/test/java/edu/ucsb/cs156/happiercows/controllers/CommonsControllerTests.java

```diff
@@ -1235,6 +1238,28 @@ public void UpdateCommonsTest_withBoundaryParameters() throws Exception {
         assertEquals(100, commonsPlus.getEffectiveCapacity());
     }
 
+    @WithMockUser(roles = {"USER"})
+    @Test
+    public void getDefaultsTest() throws Exception {
+        Commons commons = Commons.builder()
```

Comment:
minor style thing, perhaps use correct AAA ordering? i.e., move line 1260 to before getting response and maybe add comments separating arrange/act/assert. not that important since none of the other tests use this but it's good practice

Reply:
Good feedback!